packages/components/inputs/localized-rich-text-input/src/editor.tsx

```diff
@@ -44,10 +54,45 @@ const Row = styled.div`
   justify-content: flex-end;
 `;
 
-const Editor = (props) => {
+export type TEditorProps = {
+  children: ReactNode;
+  id?: string;
+  isOpen: boolean;
+  isDisabled?: boolean;
+  isReadOnly?: boolean;
+  hasWarning?: boolean;
+  hasError?: boolean;
+  editor?: TEditor;
```

Comment:
perhaps it's worth marking this prop as required. It would help out with e.g. `editor={props.editor as TEditor}` in line 185 and seems this is what `RichTextBody` is expecting really

Reply:
That means every other component dependent on the Editor component will have the editor prop as required also. 
I don't think that is the intended functionality.